audinterface/core/feature.py

```diff
@@ -519,7 +523,7 @@ def process_index(
             root: str = None,
             cache_root: str = None,
     ) -> pd.DataFrame:
-        r"""Extract features from an index conform to audformat_.
```

Comment:
Why did you remove the link?

Reply:
I did remove it as in the summary table that `autosummary` creates (which we no longer use here, but which was used in #85), the link does not work. I think in general it is a good idea if the first line of the docstring works without extra input. But I'm also fine with adding the link back.

Reply:
I would leave it as it might not be obvious for a user what `audformat` is.

Reply:
I re-enabled the links in all docstrings.